metadata-ingestion/examples/library/create_datajob.py

```diff
@@ -1,12 +1,18 @@
-from datahub.metadata.urns import DataFlowUrn
-from datahub.sdk import DataHubClient, DataJob
+from datahub.sdk import DataFlow, DataHubClient, DataJob
 
 client = DataHubClient.from_env()
 
-# datajob will inherit the platform from the flow_urn
+# datajob will inherit the platform and platform instance from the flow
+
+dataflow = DataFlow(
+    platform="airflow",
+    name="example_dag",
+    platform_instance="PROD",
```

Comment:
let's make these examples be a bit more interesting? e.g. have some tags / description / other details maybe?

Reply:
ahh i thought we supposed to provide the simplest way to init entity, updated